autotest/setup.cfg

```diff
@@ -0,0 +1,65 @@
+[flake8]
+ignore = 
+    E101  # Indentation contains mised spaces and tabs
+    E111  # Indentation is not a multiple of 4
+    E114  # Indentation is not a multiple of 4 (comment)
+    E115  # Expected an indented block (comment)
+    E117  # Over-indented comment
+    E121  # Continuation line under-indented for hanging indent
+    E122  # Continuation line missing indentation or outdented
+    E123  # Closing bracket does not match indentation of opening bracket
+    E124  # Closing bracket does not match visual indentation
+    E125  # Continuation line with same indent as next logical line
+    E126  # Continuation line over-indented for hanging indent
+    E127  # Continuation line over-indented
+    E128  # Continuation line under-indented
+    E131  # Continuation line unaligned for handing indent
+    E201  # Whitespace after bracket
+    E202  # Whitespace before bracket
+    E203  # Whitespace before ','
+    E211  # Whitespace before bracket
+    E221  # Multiple spaces before operator
+    E222  # Multiple spaces after operator
+    E225  # Missing whitespace around operator
+    E226  # Missing whitespace around arithmetic operator
+    E227  # Missing whitespace around bitwise or shift operator
+    E228  # Missing whitespace around modulo operator
+    E231  # Missing whitespace after ','
+    E241  # Multiple spaces after ','
+    E251  # Unexpected spaces around keyword equals
+    E252  # Missing whitespace around parameter equals
+    E261  # At least two spaces before inline comment
+    E262  # Inline comment should start with '# '
+    E265  # Block comment should start with '# '
+    E266  # Too many leading '#' for block comment
+    E271  # Multiple spaces after keyword
+    E301  # Expected 1 blank line, found 0
+    E302  # Expected 2 blank lines, found 1
+    E303  # too many blank lines
+    E305  # Expected 2 blank lines after class or function definition
+    E306  # Expected 1 blank line before a nested definition
+    E402  # Module level import not at top of file
+    E501  # Line too long
+    E502  # Backslash is redundant between brackets
+    E703  # Statement ends with a semicolon
+    E711  # Comparison to None should be 'if cond is None'
+    E712  # Comparison to False should be 'if cond is False' or 'if not cond'
+    E713  # Test for membership should be 'not in'
+    E721  # Do not compare types, use isinstance()
+    E722  # Do not use bare 'except'
+    E731  # Do not assign a lambda expression, use a def
+    E741  # Ambiguous variable name 'l'
+    E291  # Trailing whitespace
+    F401  # Module imported but unused
```

Comment:
F401 is currently selected, and should not be in the ignore list. Same for F403, F405, F811, F841

Reply:
Ah, I created the list by assuming any current failures were meant to be accepted, but changing the paths in code_checks.yml fired off a number of warnings in gdal/swig/python/osgeo/.

Should all files in gdal/swig/python/osgeo/ be linted? The old code_checks.yml pointed to a pair of nonexistent directories in there - utils/ and utils/auxiliary.

Reply:
Ah, the files auto-generated by Swig definitely shouldn't be linted.

Reply:
> The old code_checks.yml pointed to a pair of nonexistent directories in there - utils/ and utils/auxiliary.

there has been some file moving in last months, and code_checks.yml has obviously failed to catch those moves. I guess @idanmiara could help  if needed in linting the issues that have been introduced lately

Reply:
> Ah, the files auto-generated by Swig definitely shouldn't be linted.

agreed

Reply:
> > The old code_checks.yml pointed to a pair of nonexistent directories in there - utils/ and utils/auxiliary.
> 
> there has been some file moving in last months, and code_checks.yml has obviously failed to catch those moves. I guess @idanmiara could help if needed in linting the issues that have been introduced lately

Sure, please ping me if you need help with that.